docker/development/docker-compose-postgres.yml

```diff
@@ -49,6 +49,7 @@ services:
       - discovery.type=single-node
       - ES_JAVA_OPTS=-Xms1024m -Xmx1024m
       - plugins.security.disabled=true
+      - OPENSEARCH_INITIAL_ADMIN_PASSWORD=OpenMetadata_password123!!!
     networks:
```

Comment:
@sushi30 curious why we need these here?

Reply:
the docker opensearch docker container failed to start without setting a strong password